styles/index.css

```diff
@@ -0,0 +1,65 @@
+body {
```

Comment:
I know we talked about using the grid to overlap elements. Here is a resource to do that https://www.joomlashack.com/blog/tutorials/layering-items-within-css-grid/